python/python/glide/async_commands/core.py

```diff
@@ -694,6 +694,29 @@ async def lpush(self, key: str, elements: List[str]) -> int:
             int, await self._execute_command(RequestType.LPush, [key] + elements)
         )
 
+    async def lpushx(self, key: str, elements: List[str]) -> int:
+        """
+        Inserts specified values at the head of the list, only if `key` already exists and holds a list.
+
+        See https://redis.io/commands/lpushx/ for more details.
+
+        Args:
+            key (str): The key of the list.
+            elements (List[str]): The elements to insert at the head of the list stored at `key`.
+
+        Returns:
+            int: The length of the list after the push operations.
```

Comment:
What is better?
```suggestion
            int: The length of the list after the push operation.
```

Reply:
I think either is fine. In Java, lpush/lpushx/rpush/rpushx say "operation", in Python lpush/rpush say "operations" so I followed that pattern